docs/ja/sql-reference/sql-statements/keywords.md

```diff
@@ -70,6 +70,7 @@ SQL ステートメント内のキーワード、例えば `CREATE` や `DROP` 
 - CURRENT_DATE
 - CURRENT_TIME
 - CURRENT_TIMESTAMP
+- CURRENT_TIMEZONE
```

Comment:
CURRENT_TIMEZONE You did not add it to the keywords syntax, so the document here does not need to be modified.